src/subcommands/run.ts

```diff
@@ -19,13 +19,14 @@ USAGE:
     deployctl run [OPTIONS] <ENTRYPOINT>
 
 OPTIONS:
-        --addr=<addr>   The address to listen on (default ":8080")
-    -h, --help          Prints help information
-        --inspect       Activate inspector on 127.0.0.1:9229
-        --libs=<libs>   The deploy type libs that are loaded (default "ns,window,fetchevent")
-        --no-check      Skip type checking modules
-    -r, --reload        Reload source code cache (recompile TypeScript)
-        --watch         Watch for file changes and restart process automatically
+        --addr=<addr>         The address to listen on (default ":8080")
+    -h, --help                Prints help information
+        --inspect             Activate inspector on 127.0.0.1:9229
+        --libs=<libs>         The deploy type libs that are loaded (default "ns,window,fetchevent")
+        --no-check            Skip type checking modules
+        --env=<path/to/env>   Load envirnoment variables from the provided .env file
```

Comment:
Can you move between --addr and --help (sort by alphabet), and change `path/to/env` to `path/to/.env`?